CI.hs

```diff
@@ -62,30 +74,33 @@ parseOptions = Options
        "ghc-master" -> Right GhcMaster
        flavor -> Left $ "Failed to parse ghc flavor " <> show flavor <> " expected ghc-8.8.1 or da-ghc-8.8.1"
 
-
 bintrayUpload :: String -> IO ()
 bintrayUpload version = do
     credentials <- Env.lookupEnv "BINTRAY_BASIC_AUTH"
     case credentials of
       Nothing ->
-        Exit.die $ unlines ["Error: Cannot upload without BINTRAY_BASIC_AUTH.",
-                            "To set the environment variable, run:",
-                            "    BINTRAY_BASIC_AUTH=<creds> ./CI.hs --upload-to-bintray",
-                            "where <creds> should be of the form:",
-                            "  fname.lname@digitalassetsdk:0123456789abcdef0123456789abcdef01234567",
-                            "You can find your API key (the part after the colon) at:",
-                            "  https://bintray.com/profile/edit",
-                            "after logging in. (The username is also displayed on that page.)"]
+        Exit.die $ unlines [
```

Comment:
It would be nice if you could separate reformatting from actual changes. Mixing the two in the same PR makes it a bit hard to see what actually changed in a PR.

Reply:
Yeah. Sorry! 🙇